misk-admin/web/tabs/web-actions/src/rewrite/WebActionSendRequestFormComponents.tsx

```diff
@@ -126,7 +126,7 @@ function FormEnumComponent({
   onChange
 }: FormComponentProps<string>) {
   const enumValues = field.type
-    .replace(">", "")
+    .replace(/>/g, "")
```

Comment:
I'm not familiar with this syntax and the tests for the admin dashboard tabs are insufficient to merge confidently. 

Have you tried building the tab locally and then running the exemplar service locally to confirm it still works as expected?

Reply:
Just ran this locally and it works fine. 

Without the change:
<img width="562" alt="Screenshot 2024-07-03 at 4 34 27 PM" src="https://github.com/cashapp/misk/assets/54708104/afae5228-2d8a-43b3-87d2-e18c553b99eb">

With change:
<img width="562" alt="Screenshot 2024-07-03 at 4 48 45 PM" src="https://github.com/cashapp/misk/assets/54708104/dfa95b47-4a63-4e60-a567-c87124f6b530">